ts/Dashboards/Components/HighchartsComponent/HighchartsComponent.ts

```diff
@@ -396,7 +396,7 @@ class HighchartsComponent extends Component {
      */
     private setOptions(): void {
         if (this.options.chartClassName) {
-            this.chartContainer.classList.add(this.options.chartClassName);
+            this.chartContainer.classList.value = this.options.chartClassName;
```

Comment:
Won't this remove any other class names that should remain and not be replaced completely?